### DISCUSSION TITLE
Form Pembayaran & Renew KirimWA

### DISCUSSION THREAD

**Discussion by sifuim** [kirimwa-id/forum#183] [Saran dan Ide]
Hi,
Saya syorkan buat 1 form untuk pengguna melanggan atau renew kirimwa punya subscription. seperti sekarang, saya sudah emelkan dan mahu renew, tetapi tiada maklum balas dari pihak tuan.

saya juga ada mengsyorkan kepada client dan student di Malaysia. tetapi jika proses langganan agak rumit begini. Jadi susah untuk melanggan.

Mohon adakan pembayaran melalui stripe atau paypal.
Mohon sangat-sangat cepat diselesaikan.

**Comment by udien-arrow** [comment]
untuk yg payment local bisa lewat qris atau bank transfer mungkin mas @rioastamal

**Reply by rioastamal** [reply]
@sifuim Mohon maaf atas email yang tidak dibalas. Untuk form pembayaran/invoice sedang disiapkan.
@udien-arrow Terima kasih, kedepan rencana pakai QRIS. Untuk bank transfer sudah dari lama, pelanggan Enterprise berlangganan menggunakan pembayaran bank transfer.

**Reply by arhaviolet** [reply]
@rioastamal mas apa ada invoiced jika pembayaran via transfer?
